app/script_variant.py

```diff
@@ -145,6 +145,53 @@ def romanize_word_by_unidecode(word):
     romanized_word = unidecode(word)
     return [romanized_word.strip()]
 
+def is_chinese_person_name(word):
+    """
+    This method checks if a unicode string is a chinese person name by figuring out
+    if it contains valid a chinese surname and a given name.
+    
+    Return:
+        Boolean
+    """
+    surname, lastname  = split_chinese_name(word)
+    return surname is not None and lastname is not None
+
+def split_chinese_name(word):
+    """
+    This method tries to split a chinese name into two parts: surname and lastname
+    Args:
+        word: a chinese name string
+    Returns:
+        surname, lastname if it is a valid chinese name
+        else None, None
+    """
+    word = word.replace(" ", "")
+    if len(word) != len(re.findall(ur"[\u4e00-\u9fa5]", word)):
+        return None, None
+
+    if word[:2] in CHINESE_FAMILY_NAME_DICTIONARY:
+        return word[:2], word[2:]
+
+    elif word[0] in CHINESE_FAMILY_NAME_DICTIONARY:
+        return word[0], word[1:]
+
+    return None, None
+
+def romanize_chinese_person_name(word):
+    """
+    This method romanizes a chinese person name including surname and last name.
+
+    Returns:
+        romanized_cn_name
+    """
+
+    surname, lastname = split_chinese_name(word)
+    if not surname or not lastname:
+        return []
+
+    romanized_surname_list = list(CHINESE_FAMILY_NAME_DICTIONARY[surname])
+    return romanized_surname_list[0].capitalize() + unidecode(lastname).strip()
```

Comment:
I noticed that original unidecode will output capitalised results so I added `capitalize` here to avoid potential problems. Perhaps it is not necessary.